app/helpers/json_ld_helper.rb

```diff
@@ -155,24 +155,49 @@ def safe_for_forwarding?(original, compacted)
     end
   end
 
-  def fetch_resource(uri, id_is_known, on_behalf_of = nil, request_options: {})
+  # Fetch the resource given by uri.
+  # @param uri [String]
+  # @param id_is_known [Boolean]
+  # @param on_behalf_of [nil, Account]
+  # @param raise_on_error [Boolean, Symbol<:all, :temporary>] See {#fetch_resource_without_id_validation} for possible values
+  def fetch_resource(uri, id_is_known, on_behalf_of = nil, raise_on_error = false, request_options: {})
```

Comment:
previously, `fetch_resource_without_id_validation` had the `raise_on_temporary_error` parameter but this parent `fetch_resource` method didn't allow it to be passed through, which seemed like an accidental omission in api consistency, so i added it, though since we call it with `id_is_known = true` i could just call `fetch_resource_without_id_validation` by itself without changing this.

Reply:
Minor thing but I'd have `raise_on_error` as a keyword argument.

Reply:
aha, didn't know that was a different thing in ruby, used to python. done in https://github.com/mastodon/mastodon/pull/32615/commits/729bc9f47fd669cc592e96bea70a43f3e72d556a